en/docs/reference/customize-product/extending-api-manager/extending-gateway/writing-custom-handlers.md

```diff
@@ -37,10 +37,6 @@ Let's see what each handler does:
  the token is of type `Production` or `Sandbox` and sets `MessageContext` variables as appropriate.
  - **APIThrottleHandler:** Throttles requests based on the throttling policy specified by the `policyKey` property. 
  Throttling is applied both at the application level as well as subscription level.
- - **APIMgtUsageHandler:** Publishes events to WSO2 Stream Processor (WSO2 SP) for collection and analysis of statistics. 
```

Comment:
We have a new handler in APIM analytics. I think we have to add that here replacing this one

Reply:
New handler can be found here https://github.com/wso2/carbon-apimgt/blob/master/components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/handlers/analytics/AnalyticsMetricsHandler.java

Reply:
@ruks - Can you update this PR or type in the description for this handler?

Reply:
Will send a new PR for this.
Will merge this PR now.